promql/functions.go

```diff
@@ -1367,7 +1367,7 @@ func init() {
 	// with xrate or yrate functions. This allows for a drop-in replacement and Grafana
 	// auto-completion, Prometheus tooling, Thanos, etc. should still work as expected.
 	switch os.Getenv("REPLACE_RATE_FUNCS") {
-	case "1":
+	case "1", "x", "X":
```

Comment:
Is it a good idea to support both upper and lowercase here? I'm a bit tempted to drop the uppercase, just to have fewer ways of doing the same thing. What do you think @cmparkinson ?

Reply:
I don't mind it, seems reasonable to me.

Reply:
You don't mind both casings? Or don't mind if I just standardize on `x` and `y`? The latter is my leaning. Slightly.

Reply:
Hmm, possibly tie breaker the other way: `x` and `y` look really similar to `X` and `Y`. Lots of room for confusion there. If I only allow one, the docs probably need to call out the casing. Too much trouble.

Reply:
>You don't mind both casings?

Yeah, both seems good to me. I think I prefer handling both cases.